seerep_msgs/protos/label.proto

```diff
@@ -5,5 +5,7 @@ package seerep.pb;
 message Label
 {
   string label = 1;
-  float confidence = 2;
+  int32 labelIdDatumaro = 2;
+  string instanceUuid = 3;
+  int32 instanceIdDatumaro = 4;
```

Comment:
It's generally not recommended to use `int32` or `int64` as for IDs https://protobuf.dev/programming-guides/api/#integer-field-for-id, but instead use a `string`. However, I can see that it's very unlikely that we will get IDs larger than $2^{32}-1$ in the datumaro json.